bin/install-studio-cli.sh

```diff
@@ -1,18 +1,18 @@
 #!/bin/sh
 
-# This script is used to install the Studio CLI on macOS. It creates a symlink at CLI_SYMLINK_SOURCE
-# (e.g. /usr/local/bin/studio) pointing to the packaged Studio CLI JS file at CLI_SYMLINK_TARGET.
+# This script is used to install the Studio CLI on macOS. It creates a symlink at CLI_SYMLINK_PATH
+# (e.g. /usr/local/bin/studio) pointing to the packaged Studio CLI JS file at CLI_PACKAGED_PATH.
```

Comment:
`CLI_SYMLINK_PATH` and `CLI_PACKAGED_PATH` are great names 👍
Similar to what I proposed [( CLI_SYMLINK_PATH / CLI_TARGET_PATH )](https://github.com/Automattic/studio/pull/1141/files), but `CLI_PACKAGED_PATH` is much clear 👍